Description: Fix FTBFS on ppc64el
Fix FTBFS on ppc64el:
  - include ppu_intrinsics.h to resolve __fsel
  - remove register from non-registerable variables
Author: Andreas Barth <aba@ayous.org>
Bug-Debian: https://bugs.debian.org/763691

--- cdo-1.6.4+dfsg.1.orig/libcdi/src/cgribexlib.c
+++ cdo-1.6.4+dfsg.1/libcdi/src/cgribexlib.c
@@ -5,6 +5,7 @@
 
 #ifdef _ARCH_PWR6
 #pragma options nostrict
+#include <ppu_intrinsics.h>
 #endif
 
 #if defined (HAVE_CONFIG_H)
@@ -908,8 +909,8 @@ void pwr6_minmax_val_unrolled6(const dou
     size_t i, j;
     size_t residual =  datasize % __UNROLL_DEPTH_1;
     size_t ofs = datasize - residual;
-    double register dmin[__UNROLL_DEPTH_1];
-    double register dmax[__UNROLL_DEPTH_1];
+    double dmin[__UNROLL_DEPTH_1];
+    double dmax[__UNROLL_DEPTH_1];
 
     for ( j = 0; j < __UNROLL_DEPTH_1; j++) 
       {
